src/SFA.DAS.EmployerIncentives.Api.AcceptanceTests/Features/IncentiveApplicationCreated.feature

```diff
@@ -0,0 +1,11 @@
+﻿@database
+@api
+Feature: IncentiveApplicationCreated
+	In order to select apprenticeships to apply for an incentive for
+	As an employer
+	I want to know if an apprenticeship is eligible
+
+Scenario: Incentive Application is created
+	Given I am applying for the New Apprenticeship Incentive
```

Comment:
It's generally better to write in the third person because it's not clear who or what is "I"?  Is it an employer, and apprentice a provider a support user?  So better to use An employer applying for ...